tools/hidl/hidlcomp.cpp

```diff
@@ -5800,7 +5807,7 @@ void EspServInfo::write_esp_binding()
             if (hasMinVer)
             {
                 outf("\t\tif (clientVer!=-1.0 && clientVer<%s)\n", minVer.str());
-                outs("\t\t\tthrow MakeStringException(-1, \"Client version is too old (can't pass exception to client)\");\n");
+                outf("\t\t\tthrow MakeStringException(-1, \"This method is not supported in version %%g, minimum version is %s. Please update your client application.\", clientVer);\n", minVer.str());
```

Comment:
Fyi, trivial, but you could also use the indent parameter here as well.

Reply:
Thanks, @afishbeck. I chose to use \t here because that's what's written in the lines above and below it. Should I change it?

Reply:
Yes, I realized that's why you did it and most of the existing code is that way.  But we might as well improve it as we go.  So, yes, if you don't mind, might as well change it while we are here.

Reply:
OK, I've submitted a new commit changing the entire block.